analyzers/tests/SonarAnalyzer.UnitTest/AnalysisContext/SonarAnalysisContextBaseTest.ShouldAnalyzeTree.cs

```diff
@@ -146,18 +141,18 @@ public void ShouldAnalyzeTree_GeneratedFile_AnalyzeGenerated_AnalyzeAllFiles(str
     public void ShouldAnalyzeTree_CorrectSettingUsed_VB(string fileName, bool expectedCSharp)
     {
         var sonarLintXml = CreateSonarLintXml(false);
-        var options = CreateOptions(sonarLintXml);
         var (compilationCS, treeCS) = CreateDummyCompilation(AnalyzerLanguage.CSharp, fileName);
         var (compilationVB, treeVB) = CreateDummyCompilation(AnalyzerLanguage.VisualBasic, fileName);
-        var sut = CreateSut();
+        var sutCS = CreateSut(compilationCS, CreateOptions(sonarLintXml));
+        var sutVB = CreateSut(compilationVB, CreateOptions(sonarLintXml));
```

Comment:
After the changes both cs and vb are tested but the test was meant only for VB. You need to update the test name or better extract another test for C# if needed.